pkg/trace/schema/consensus.go

```diff
@@ -128,3 +129,54 @@ func WriteBlock(client *trace.Client, block *types.Block, size int) {
 		LastCommitRoundFieldKey:          block.LastCommit.Round,
 	})
 }
+
+// Schema constants for the consensus votes tracing database.
+const (
+	// VoteTable is the name of the table that stores the consensus
+	// voting traces. Follows this schema:
+	//
+	// | time | height | round | vote_type | vote_height | vote_round
```

Comment:
The height and round of the vote may not always correspond with the sending peer's round and height. Therefore, the round and height of the sending peer are tracked in separate fields. This information is valuable as it provides insight into the state of peers from each other's perspective.

Reply:
nice, I like that a lot